dll/win32/shell32/wine/shellord.c

```diff
@@ -1386,10 +1386,22 @@ HRESULT WINAPI SHWinHelp(HWND hwnd, LPCWSTR pszHelp, UINT uCommand, ULONG_PTR dw
  *  SHRunControlPanel [SHELL32.161]
  *
  */
-BOOL WINAPI SHRunControlPanel (LPCWSTR commandLine, HWND parent)
+BOOL WINAPI SHRunControlPanel (_In_ LPCWSTR commandLine, _In_opt_ HWND parent)
 {
+#ifdef __REACTOS__
+    //TODO: Run in-process when possible, (HKLM\Software\Microsoft\Windows\CurrentVersion\Explorer\ControlPanel\InProcCPLs), and possibly some extra rules,
+    //TODO: 'If the specified Control Panel item is already running, SHRunControlPanel attempts to switch to that instance rather than opening a new instance.'
+    //TODO: This function is not supported as of Windows Vista and as of Windows Vista, this function always returns FALSE (https://docs.microsoft.com/en-us/windows/win32/api/shlobj/nf-shlobj-shruncontrolpanel )
+    //but we need to keep it "live" even when reactOS is compliled as NT6+ in order to keep control panel elements launch commands.
```

Comment:
```suggestion
    /*
     * TODO: Run in-process when possible, using
     * HKLM\Software\Microsoft\Windows\CurrentVersion\Explorer\ControlPanel\InProcCPLs
     * and possibly some extra rules.
     * See also https://docs.microsoft.com/en-us/windows/win32/api/shlobj/nf-shlobj-shruncontrolpanel
     * "If the specified Control Panel item is already running, SHRunControlPanel
     *  attempts to switch to that instance rather than opening a new instance."
     * This function is not supported as of Windows Vista, where it always returns FALSE.
     * However we need to keep it "alive" even when ReactOS is compliled as NT6+
     * in order to keep control panel elements launch commands.
     */
```